src/mediapipe_internal/mediapipegraphdefinition.hpp

```diff
@@ -147,6 +155,8 @@ class MediapipeGraphDefinition {
     std::vector<std::string> inputSidePacketNames;
 
     std::atomic<uint64_t> requestsHandlesCounter = 0;
+
+    std::map<std::string, std::unique_ptr<NodeState>> pythonNodeStates;
```

Comment:
Use unordered_map when you do not care about key ordering

Reply:
std::map was used in design.